ckanext/gdi_userportal/validation.py

```diff
@@ -0,0 +1,134 @@
+# SPDX-FileCopyrightText: His Majesty the King in Right of Canada, represented by the President of the Treasury Board
+# SPDX-FileContributor: Stichting Health-RI
+#
+# SPDX-License-Identifier: MIT
+
+from datetime import datetime
+
+import pytz
+from ckanext.scheming.validation import register_validator, scheming_validator
+from ckantoolkit import Invalid, _, get_validator
+from dateutil.parser import isoparse
+
+not_empty = get_validator("not_empty")
+
+
+def enforce_utc_time(dt: datetime) -> datetime:
```

Comment:
This is a more clean way of enforce UTC time. @a-nayden  choose to extend the date preset with time. Could you remove the code in the fairdatapoint?